backend/wmg/data/load.py

```diff
@@ -0,0 +1,19 @@
+def update_s3_resources():
+    """
+    Copy cube and cube related files to s3 under the current timestamp
+    """
+    pass
+
+
+def remove_oldest_data():
```

Comment:
```suggestion
def remove_previous_snapshot():
```

Reply:
Or maybe it takes a timestamp, since it might be safer if the caller decides which snapshot to remove. can then then rename to just `remove_snapshot(timestamp)`

Reply:
Maybe we need a name for all of the data (I intended this to remove the expression summary cube, the cell count cube, the corpus and the cell ordering json file)

Reply:
My understanding was that snapshot just referred to the expression summary cube -- let me know if thats incorrect